R/animint.R

```diff
@@ -130,11 +133,12 @@ parsePlot <- function(meta){
     ## Data now contains redundant columns with fill, alpha, colour etc.
     ## Remove from data if they have a single unique value and
     ## are NOT used in mapping to reduce tsv file size
+    redundant.cols <- c("fill", "shape", "color", "colour", "stroke", "alpha",
+                        "linetype", "size", "angle", "vjust", "family",
+                        "fontface", "lineheight", "anchor")
```

Comment:
@tdhock I just thought it would be safer to remove the ones listed here. Ideally, it should work without specifying the columns too. But I specified them to be on the safe side.